Lib2025/src/main/java/frc/robot/commands/intake/WaitForBeamBreakState.java

```diff
@@ -0,0 +1,11 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Robot;
+
+public class WaitForBeamBreakState extends Command {
```

Comment:
For this command, we'll want to be able to pass in the state we want to wait for then the isFinished will return whether `isBeamBroken` is equal to that state.